django/core/management/commands/makemessages.py

```diff
@@ -615,13 +615,15 @@ def write_po_file(self, potfile, locale):
 
         if os.path.exists(pofile):
             args = ['msgmerge'] + self.msgmerge_options + [pofile, potfile]
-            msgs, errors, status = popen_wrapper(args)
+            _, errors, status = popen_wrapper(args)
             if errors:
                 if status != STATUS_OK:
                     raise CommandError(
                         "errors happened while running msgmerge\n%s" % errors)
                 elif self.verbosity > 0:
                     self.stdout.write(errors)
+            with open(pofile, encoding='utf-8') as fp:
+                msgs = fp.read()
```

Comment:
I think we can use pathlib for new code.
```suggestion
            msgs = Path(pofile).read_text(encoding='utf-8')
```